content/src/fieldConfig/business-formation.json

```diff
@@ -265,7 +265,7 @@
       "businessStartDate": {
         "label": "NJ Business Effective Date",
         "labelContextualInfo": "effective-date",
-        "labelSecondaryText": "(mm/dd/yyyy)",
+        "labelSecondaryText": "(mm/dd/yyyy) Eastern Time",
```

Comment:
eastern date?

Reply:
This is what Content and Design asked for here.

Reply:
"Eastern Time" would actually be the most acceptable name here for the timezone, even though you're only specifying the date.